pkg/innerring/processors/netmap/cleanup_table.go

```diff
@@ -64,6 +68,14 @@ func (c *cleanupTable) update(snapshot netmap.NetMap, now uint64) {
 	}
 
 	c.lastAccess = newMap
+
+	// order is expected to be the same from epoch to epoch
```

Comment:
can u pls remind what ensures this?

Reply:
netmap are candidates that are stored in storage by their pub key, so it is sorted for sure. then it is `std.Serialize`. @roman-khimov, is array serialization in vm stable?

Reply:
VM is stable in every aspect of it given that inputs/outputs are the same.